### PR TITLE
Zeek dns.log

### PR DESCRIPTION
**Pull request by rekidderjr**
Looking for review of proposed Zeek dns.log mapping

### CODE REVIEW

File: Network Activity/DNS Activity/Zeek/dns_log/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+| OCSF                       | Raw             |
+| -------------------------- | ----------------|
+|'trans_id'|unmapped'|
+|'rtt'|unmapped'|
+|'AA'|unmapped'|
+|'TC'|unmapped'|
+|'RD'|unmapped'|
+|'RA'|unmapped'|
+|'Z'|unmapped'|
+|'TTLs'|unmapped'|
```
Comment (floydtree) [inline Network Activity/DNS Activity/Zeek/dns_log/README.md:57]: can be mapped to `dns_answer.ttl`

**Review by floydtree — CHANGES_REQUESTED** [review]
Can you also add a transformed/OCSF version of the zeek event that you have?

File: Network Activity/DNS Activity/Zeek/dns_log/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ - `severity_id`: `1`
+
+ ### Mapping:
+ - This does not reflect any transformations or evaluations of the data. Some data evaluation and transformation will be necessary for a correct representation in OCSF that matches all requirements.
+
+ - Any field not present in an explicit mapping will be moved to the unmapped object.
+
+| OCSF                       | Raw             |
+| -------------------------- | ----------------|
+|'time'|'time'|
```
[the remark this answers is not in the document]
Comment (rkidderaws) [reply Network Activity/DNS Activity/Zeek/dns_log/README.md:27]: Sorry I miss pasted both raw and ocsf with the same values.  Whoops! Fixing now

File: Network Activity/DNS Activity/Zeek/dns_log/README.md (hunk 1 above)
Comment (rkidderaws) [reply Network Activity/DNS Activity/Zeek/dns_log/README.md:57]: Corrected ttl mapping as requested.

**Pull request closed without merging**